src/ProjectService.Models.Dto/Requests/Filters/GetDbProjectsUserFilter.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace LT.DigitalOffice.ProjectService.Models.Dto.Requests.Filters
+{
+    public class GetDbProjectsUserFilter
+    {
+        public Guid? UserId { get; set; }
```

Comment:
Why nullable now?

Reply:
Because of it find now, we can filter by other properties